portfolio/src/main/webapp/script.js

```diff
@@ -25,44 +25,58 @@ function addRandomFact() {
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
+  factContainer.style.visibility = 'visible';
 }
 
 /**
  * Creates a semi-random story and adds it to the page.
  */
 function randomizeStory() {
-    var inputFromUser = document.getElementById('inputName').value;
-    var inputName = (inputFromUser == '') ? "Karen" : inputFromUser;
-    var objectsList = ["calculator", "adventure book", "arrow", "wand", "ear plugs"]
-    var characterList = ["snail", "witch", "king", "rabbit", "lion", "clown"]
-    var text = "";
+  var inputFromUser = document.getElementById('inputName').value;
```

Comment:
Note: `var` is an old-style declaration of JS variables.
Prefer to use `const` or `let` for new variables. They have block scope (rather than var that has a function scope); And `const` variables are constant throughout its lifetime.

Reply:
got it!